commit/fees/types.go

```diff
@@ -0,0 +1,20 @@
+package fees
+
+import (
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+)
+
+type Query struct {
+}
+
+type Outcome struct {
+	GasPrices []cciptypes.GasPriceChain `json:"gasPrices"`
+}
+
+type Observation struct {
+	GasPrices []cciptypes.GasPriceChain `json:"gasPrices"`
+}
+
+func (o Outcome) Sort() {
+	// TODO: implement
+}
```

Comment:
I feel like we should just enforce that these outcomes that are produced are always correctly sorted in tests.

Reply:
Having outcome implement it makes it clear that we need to sort them. I agree we need to test it as well. I'm going to use the sort inside the Outcome function in each of the sub plugins

Reply:
Don't think I agree with that, test enforcement is always better than this kind of implicit contract ("plz implement")

Reply:
I don't see the conflict. What's the problem with having both?